### PR TITLE
Update data_analysis.md

### PR DESCRIPTION
**Pull request by OlivierCollin**
First version of the "Data Analysis" page.

### CODE REVIEW

File: pages/your_problem/data_analysis.md [hunk 1]
```diff
@@ -1,39 +1,65 @@
 ---
 title: Data Analysis
 keywords: optional
-contributors: []
-tags: []
+contributors: [Olivier Collin]
+tags: [Data analysis]
```
Comment (bedroesb) [inline pages/your_problem/data_analysis.md:5]:
```suggestion
tags: [analyse]
```
In order for this page to show up in the Data life Cycle page Analysing, it need to be tagged with analyse.

**Review by bedroesb — COMMENTED** [review]
Thank you for starting this!

File: pages/your_problem/data_analysis.md (hunk 1 above)
Comment (OlivierCollin) [reply pages/your_problem/data_analysis.md:5]: Good idea ! 
I completely forgot the link with the Data life Cycle....

**Review by floradanna — CHANGES_REQUESTED** [review]
Thanks for this contribution. Please, make sure that all tools and resources mentioned are added to the tool and resource table as described in the [guide](https://rdm.elixir-europe.org/tool_resource_update.html#making-changes).

Git
Conda
Bioconda
Docker
Singularity
Galaxy
Nextflow
Snakemake
Jupyter notebook
RStudio

**Comment by YvonneKallberg** [comment]
I would remove space previous to ':' and '?':
- What are the best practices for data analysis ?
- You can act at several levels :
- One can mention :
- Among the most well-known notebooks systems are :

**Comment by OlivierCollin** [comment]
> Thanks for this contribution. Please, make sure that all tools and resources mentioned are added to the tool and resource table as described in the [guide](https://rdm.elixir-europe.org/tool_resource_update.html#making-changes).
> 
> Git
> Conda
> Bioconda
> Docker
> Singularity
> Galaxy
> Nextflow
> Snakemake
> Jupyter notebook
> RStudio

Thank you Flora for your feedback. 
I have added the missing tools in the spreadsheet.

**Pull request merged**